### PR TITLE
update ci

### PR DESCRIPTION
**Pull request by byronz**
the change in the CI config should fix the problem where a test case failed, percy cannot be finalized properly, i.e. a simple rerun will introduce a 409 error.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -189,6 +189,7 @@ jobs:
       - image: circleci/python:3.7-stretch-node-browsers
         environment:
           PERCY_PARALLEL_TOTAL: -1
+          PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: True
```
Comment (alexcjohnson) [inline .circleci/config.yml]: What's this about?

File: .circleci/config.yml [hunk 2]
```diff
@@ -218,6 +219,10 @@ jobs:
           path: test-reports
       - store_artifacts:
           path: /tmp/dash_artifacts
+      - run:
+          name: 🦔 percy finalize
+          command: npx percy finalize --all
+          when: on_fail
```
Comment (alexcjohnson) [inline .circleci/config.yml]: interesting... how does this work with parallelism? I'm just wondering what will happen if it fails early in one container, runs `percy finalize` from there, then in another container more snapshots are recorded. If it just silently fails to add those to the run that would be fine, but if it makes more errors in the other container that could get very confusing.

File: .circleci/config.yml (hunk 1 above)
Comment (byronz) [reply .circleci/config.yml]: the npx needs implicitly the installation of puppeteer, which will then download the big guy `chromium`, this option is to skip the download.

File: .circleci/config.yml (hunk 2 above)
Comment (byronz) [reply .circleci/config.yml]: good point about the parallelism. this step will be executed only after the full run of `pytest` in each container, no matter if it failed at an earlier stage. the whole duration is close from one to another, but you are right, it still has some risk to confuse other containers. but I think this way it's better than not having the finalize at all.

**Pull request merged**